pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "cumulus-library"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
this is fine this time, but in the future we may want to do these increments as standalone commits just to avoid merge issues - what i'm working on over on the CLI side is going to become a 0.3.0 release, for example.